web/src/beta/features/ProjectSettings/innerPages/PluginSettings/index.tsx

```diff
@@ -90,6 +90,7 @@ const PluginSettings: React.FC<Props> = ({
                     installedPlugins={marketplacePlugins}
                     onInstall={handleInstallPluginByMarketplace}
                     onUninstall={handleUninstallPlugin}
+                    version={"classic"}
```

Comment:
looks like currently we are using the `PluginSettings` from `beta` sub-folder so I made this additional change.

Reply:
```suggestion
                    version={"visualizer"}
```
This is the beta inside classic, which will be removed in the future. I think it's okey to add the prop as well but the value should be `visualizer`